frontend/app/globals.css

```
@@ -1,8 +1,28 @@
 @import "tailwindcss";
 
-/* GenLayer Brand Guidelines 2025 - Typography via next/font */
-/* --font-body: Inter (Switzer alternative) */
-/* --font-display: Space Grotesk (Lineca alternative) */
+@font-face {
+  font-family: "Switzer";
```

Stylelint (17.11.1)

[error] Expected no quotes around "Switzer" (font-family-name-quotes)

(font-family-name-quotes)